lib/beaker/testmode_switcher/runner_base.rb

```diff
@@ -0,0 +1,14 @@
+module Beaker
+  module TestmodeSwitcher
+    # Reused funtionality for all the runners
+    class RunnerBase
```

Comment:
The ruby idiom would be to use a module here, and use `include`, instead of `<` above.

This is for example used [here](https://github.com/puppetlabs/beaker-testmode_switcher/blob/master/lib/beaker/testmode_switcher/dsl.rb#L17) to implement the DSL in the global namespace.